compiler/testData/debug/stepping/nestedInline.kt

```diff
@@ -0,0 +1,63 @@
+// This is same as kotlin/compiler/testData/codegen/boxInline/smap/smap.kt
```

Comment:
I think it would be much easier to understand the LINENUMBERS data if this comment will be below the files.